update-pot

```diff
@@ -24,6 +24,14 @@ find "$HERE" -name "*.go" -type f -print0 | xargs -0 \
     --keyword=i18n.G \
     --keyword-plural=i18n.DG
 
+xgettext "$HERE"/data/polkit/*.policy \
+    -o "$OUTPUT" \
+    --no-location \
+    --package-name=snappy \
+    --msgid-bugs-address=snappy-devel@lists.ubuntu.com \
+    --join-existing \
+    || true
```

Comment:
Curious about what failure modes we see if we omit the `|| true` - if we expect this to work on all distros (can we?) then I think we should try to avoid the || true :)

Reply:
`|| true` is there only to please CI. It's not needed when I test locally, and I don't understand why CI complained with

_xgettext: input file doesn't contain a header entry with a charset specification_

in the first logs.